governance/second-generation/cloud-agnostic/http-examples/README.md

```diff
@@ -26,10 +27,17 @@ The [use-latest-module-versions.sentinel](./use-latest-module-versions.sentinel)
 * `organization` gives the name of an [organization](https://www.terraform.io/docs/cloud/users-teams-organizations/organizations.html) on the Terraform Cloud or Terraform Enterprise server specified by `address`. You must always specify a valid organization.
 * `token` gives a valid Terraform Cloud API token which can be a user, team, or organization token. See the [API tokens](https://www.terraform.io/docs/cloud/users-teams-organizations/api-tokens.html) document for more information.
 
-### Using Parameters with the Sentinel CLI
+## Use of Parameters in asteroids.sentinel
+The [asteroids.sentinel](./asteroids.sentinel) policy uses two parameters:
+* `api_token` must be set to a NASA API token. (See below.)
```

Comment:
nice use of the NASA API  👨‍🚀

Reply:
Thanks.  That was actually from a HashiCorp [blog](https://www.hashicorp.com/blog/announcing-business-aware-policies-for-terraform-cloud-and-enterprise/) when Business Aware Policies (aka, the http import) was released.  But I fixed it up  to work properly and added test cases.